### DISCUSSION TITLE
multiple pages for table

### DISCUSSION THREAD

**Discussion by HaveF** [rclement/datasette-dashboards#183] [Ideas]
For table, wouldn't it be better to set up multiple pages?
Of course, this implementation seems a bit cumbersome. Just leave an idea here.

Sometimes when layout, if the table and other components, such as the chart, are on the same line, if there is a lot of content in the table, it is easy to make the layout ugly.

As I said before, this is not a big problem, just leave an idea here.

**Comment by rclement** [comment]
Thanks @HaveF for the suggestion. I've faced this issue myself too, but I have not take some time to tackle this.

Anyway, handling this would require some client-side only pagination (JS) of the table component. It is doable but the implementation might be involved.
